server/routes/boards-routes.js

```diff
@@ -98,6 +98,7 @@ router.get('/kanban', async (req, res, next) => {
         populate: { path: 'issues', populate: { path: 'assignee' } },
       });
   } catch (e) {
+    console.log(e);
```

Comment:
Delete this since we are passing it in next()

Reply:
done